### ISSUE TITLE
Legend and Scaling

### ISSUE THREAD

**Issue by mateoclarke** [txappleseed/txappleseedmap#30]
> I also think the scale on the legend isn't very readable. Maybe we should label the left side with "less than average" and the right with "more than average" to help the readers understand what we're saying. Also, it might be better to change the left part of the scale to a negative percentage, the part that represents "no disparity" to zero (instead of one), and the right part to a positive percentage (but the positive percentage can go higher than 100%).
> @mscarey

![](http://dsh.re/68de7)

**Comment by mlsintx** [comment]
This legend is just erroneous, the overall legend.  Not sure what happened, but those are obviously whole values that need to be edited.  As for the populations, I agree that the inequity value is a little obtuse.  I think the idea of less than average  and more than average is a good one.  The inequity value was proposed by Yamanda and is explained in the header, but may still be confusing.  As for making the right side of the legend a negative number, how would that work mathematically?  The number is a ratio where 1 is equal to average, less than one is less than average and more than one is more than average.  I don't see a good way to calculate a negative number there.  This is why I proposed the divergent colormap here so the two extremes are more visible

**Comment by mscarey** [comment]
I did some work on this and added a CSV data file to [the old data directory](github.com/open-austin/texasappleseed/tree/gh-pages/data). Unfortunately it's structured pretty differently from the files currently in use, which could be inconvenient. I added a column called "Fisher" with a number between -1 and 1 to show whether a particular group is getting better or worse treatment than average that isn't likely to be a result of random chance. (I used the "Fisher's exact test" function from Scipy. If anyone can verify that the result makes sense, that would be helpful.) I was thinking the numbers in the Fisher column could correspond to colors on the map's color scale. Maybe something like warm colors for positive numbers and cool colors for negative numbers. Also, the CSV has a "disparity" column that should show the percentage difference between the number of suspensions given to a particular group of students compared to the mean. I think the "Fisher" numbers should only be used for determining the color coding on the map; they shouldn't be presented as text for users to read because they're too confusing. The percentage differences in the "disparity" column would make a lot more sense to readers.

**Comment by amaliebarras** [comment]
If we're looking at this relative to an average value, I'd recommend using diverging colors (red-green, blue-orange) where >average is one color and <average is the other. 

Like this? 

![](https://cartastrophe.files.wordpress.com/2009/08/diverging1.png?w=655)

**Comment by mscarey** [comment]
[A new version of the data file](https://github.com/open-austin/texasappleseed/blob/gh-pages/data/DistrictDisparities2014.csv) is up in the old data directory. This adds back the special education and economic disadvantage columns (which I previously deleted while trying to fix the scaling problem).

The test I used to create the scaling column assumes that each characteristic being measured can only apply to each individual once. That's not close to true here: I noticed that in many districts the special ed students averaged more than one in-school suspension per year per capita. But I'm still 90 percent sure that the numbers the computer spit out will be relevant enough to use for color scaling.

By the way, the scale will need to be logarithmic. With a range from -1 to 1, the 75th percentile is 	0.995392.
